server/p2p_receive_transaction_request_processor.go

```diff
@@ -98,15 +103,15 @@ func verifyIncomingPaymail(ctx context.Context, c *Configuration, md *RequestMet
 
 func verifySignature(metadata *paymail.P2PMetaData, txID string) error {
 	// Get the address from pubKey
-	var rawAddress *bscript.Address
+	var rawAddress *script.Address
 	var err error
 
-	if rawAddress, err = bitcoin.GetAddressFromPubKeyString(metadata.PubKey, true); err != nil {
+	if rawAddress, err = script.NewAddressFromPublicKeyString(metadata.PubKey, true); err != nil {
```

Comment:
i seen above u changed PrivateKey to PrivKey and here in the name u are using PublicKey should not be this consistent and if we use short prefix of key to Priv or Pub ?

Reply:
Unfortunately, I would like to rename `script.NewAddressFromPublicKeyString` to `script.NewAddressFromPubKeyString` but it's the method from external package. 
Then I decided to rename metadata.PubKey to metadata.PublicKey to match naming from the func.

Please tell me if it's okay.